src/servlets/SignInServlet.java

```diff
@@ -0,0 +1,41 @@
+package servlets;
+
+import templater.PageGenerator;
+import user.AccountService;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class SignInServlet extends HttpServlet {
+    private AccountService account;
+
+    public SignInServlet(AccountService account) {
+        this.account = account;
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        resp.getWriter().println(PageGenerator.instance().getPage("signin.html", account.getUserData()));
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit

Reply:
Thanks so much

Reply:
I checked it